lib/sirius/faculty_semester_weeks_generator.rb

```diff
@@ -0,0 +1,17 @@
+module Sirius
+  module FacultySemesterWeeksGenerator
+
+    module_function
+
+    def generate_semester_weeks_dates(semester)
+      faculty_semester_weeks = SiriusApi::SemesterSchedule.resolve_weeks(semester.starts_at, semester.teaching_ends_at, semester.faculty)
```

Comment:
tohle si uvědomuju, že je celkem prasárna

jak jsem slíbil, tak to pak upravím, až budu mít trochu čas... v průběhu semestru ;)